File: TsogosunProfileAdmin/MSPatronRewardsAdmin/Model/Request/RequestAddUpdateScratchCardActiveTime.cs


using MSPatronRewardsAdmin.Model.Dtos;

namespace MSPatronRewardsAdmin.Model.Request
{
    public class RequestAddUpdateScratchCardActiveTime
    {
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public ScratchCardActiveTimesDto ScratchCardActiveTimesDto { get; set; }
    }
}

File: TsogosunProfileAdminWeb/src/app/model/request/profile-rewards-admin/requestAddUpdateScratchCardActiveTime.ts
import { ScratchCardActiveTimesDto } from "../../Dtos/profile-rewards-admin/scratchCardActiveTimesDto";

export interface RequestAddUpdateScratchCardActiveTime {
    startTime?: string | undefined;
    endTime?: string | undefined;
    scratchCardActiveTimesDto?: ScratchCardActiveTimesDto | undefined;
}